docs/content/docs/try-flink-kubernetes-operator/quick-start.md

```diff
@@ -123,4 +118,10 @@ To expose the Flink Dashboard you may add a port-forward rule or start a tunnel:
 minikube tunnel
```

Comment:
`minikube tunnel` could not work now because we are using `ClusterIP` for exposing the rest service. I am suggesting to use `kubectl port-forward svc/basic-example-rest 8081`.
Also we could refer to the ingress[1] solution here.

[1]. https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/ingress/

Reply:
good catch, fixed